fletcher/pandas_compat.py

```diff
@@ -0,0 +1,13 @@
+import pandas as pd
+
+from distutils.version import LooseVersion
+
+
+def is_type_registry_available():
```

Comment:
What is the reasoning for introducing a method per feature? This currently only the checks the version and thus I see no value in adding this.

Reply:
This is just a proposal to handle the plethora of switches we have in the code base, already. imho, it makes the code better readable since one doesn't need to care about what triggers the change. Down the line I could imagine running into flags where we depend on multiple conditions and we could establish this as a best practice early on. That being said, I don't have a strong opinion on it. If you consider the indirection worthless, I can just revert to version checks wherever necessary.

Reply:
As long it is just "only available in current master", I think we should not add more complexity. We can remove all the version switches once Pandas 0.24 is released for 2-3 weeks and then impose a minimal version that matches the latest release.

Reply:
If we'll only support the most recent stable version, I agree that this complexity is unnecessary. I could imagine that some people are struggling to migrate to the most recent versions of pandas. To support multiple versions, this explicit schema could be helpful. Just like a "feature added in version x.y"
I'll remove it for now until this becomes an issue